cluster/gce/config-test.sh

```diff
@@ -156,3 +156,10 @@ HAIRPIN_MODE="${HAIRPIN_MODE:-promiscuous-bridge}" # promiscuous-bridge, hairpin
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+
+# A list of images to pre-pull for the e2e, #25277
+function get_image_list {
+  grep -Iiroh "gcr.io/google_.*" "${KUBE_ROOT}/test/e2e" | sed -e "s/[,\")}]//g" | awk '{print $1}' | sort | uniq | tr '\n' ' '
```

Comment:
We could have a whitelist with a precommit check against against what is observed to make sure it gets updated.